lib/getatrex/writer.ex

```diff
@@ -31,7 +31,48 @@ defmodule Getatrex.Writer do
     :ok = GenServer.call(__MODULE__, {:write_line, line})
   end
 
-  def handle_call({:write_message, %{mentions: mentions, msgid: msgid, msgstr: msgstr}}, _from, state)
+  def handle_call({:write_message, %{mentions: mentions, msgid: msgid, msgid_plural: msgid_plural, msgstr: msgstr, msgstr0: msgstr0, msgstr1: msgstr1, msgstr2: msgstr2, msgstr3: msgstr3, msgstr4: msgstr4, msgstr5: msgstr5}}, _from, state)
+  when is_nil(msgstr1) == false and mentions == [] or is_nil(mentions) do
+    message_string = [
+      ~s(msgid "#{msgid}"),
+      ~s(msgid_plural "#{msgid_plural}"),
+      ~s(msgstr[0] "#{msgstr0}"),
+      ~s(msgstr[1] "#{msgstr1}"),
+      if(is_nil(msgstr2), do: nil, else: ~s(msgstr[2] "#{msgstr2}")),
```

Comment:
this one could be changed set of functions that pattern-match the `msgstrN`, e.g.:

```
def get_ msgstr(nil), do: nil
def get_ msgstr(str), do: ~s(msgstr[3] "#{msgstr3}"))
```

and replace if/then/else structure into  `get_ msgstr(msgstr1)`, `get_ msgstr(msgstr2)`, etc.